### PR TITLE
Chemistry Update

### PR DESCRIPTION
**Pull request by FluffMedic**
Alright added more chems and stuff for explorery and minery folks to bring back to station.
This was also going to hide chitin and stuff for mobs but I don't know how materials work so it's just their meat.

Deathclaws, Big Dragons, Phoron Dragons, and Metroids now have meat with chems in them.

/decl/chemical_reaction/instant/livingagent
Heals but gives neagtive pain killers.

/decl/chemical_reaction/instant/preformancepeaker
Hyperzine plus but will murder you with toxins

/decl/chemical_reaction/instant/burncard
Heals 2 more then bicard, but deals consistant burn damage.

/decl/chemical_reaction/instant/flamecure
stronger myalimne but burn damage

/decl/chemical_reaction/instant/neotane
Kelotane+2 but with brute damage

/decl/chemical_reaction/instant/bloodsealer
Improaline+ with burn damage

/decl/chemical_reaction/instant/puryfingagent
I forgot which is the cheap option, but heals toxins better then dylo but worse then carth and have niether bonus upsides.
Cheap version drugs you. Also minor rad heal

/decl/chemical_reaction/instant/cleansingagent
I forgot which is the cheap option, but heals toxins better then dylo but worse then carth and have niether bonus upsides.

/decl/chemical_reaction/instant/souldew
HEALS THE DEAD

/decl/chemical_reaction/instant/quadcord
Worse tricord with another damage type to heal

/decl/chemical_reaction/instant/juggernog
The drug everyone wants, giving resistaince to stuns

/decl/chemical_reaction/instant/curea
Removes burn, poisoned, hivebot_weaken, chilled, etc modifers

/decl/chemical_reaction/instant/liquidangel
Gives pink slime effect

/decl/chemical_reaction/instant/phoneixbreath
HEALS THE DEAd and the living a tad. Stupid expensive

ALSO
Adds a special firstaid kit that gives pill version of several of this.
Is in the firstaid loot pile as a 4% drop.

### CODE REVIEW

**Comment by FluffMedic** [comment]
I didn't mess with Vr....why is it now that I am getting errors about VR...

**Comment by FluffMedic** [comment]
Alright, none of this are issues with my PRs, I don't desire to hunt down the random docking controler to fix that runtime.

**Comment by FoxglovePurpur** [comment]
Just as when you started I'm really impressed and excited for these new meds!
Adding new chems, especially ones that drop from carving up corpses, is neat. Means there's more reason to actually "hunt". I have been debating adding something to jellies to make them more med-requested, seeing as how we get them a lot but don't have much a use for them.

My only real question, seeing as how I've mostly read over commit titles and descriptions and not your code just yet, is what puryfingagent? Seeing as how the next is cleaningagent (cleaning agent) I assumed at first that it was supposed to be purifyingagent (purifying agent)?

**Comment by FoxglovePurpur** [comment]
Phoneix > Phoenix 
Expirmental > Experimental 
Orginal > Original
Juggernog's description should likely have "An experimental drug that toughens the body..." rather than just "An experimental that toughens the body..."
Is the description of living agent correct?
Neotane's descriptoin has "advancement" spelled incorrectly.
Flame cure's description spells haemorrhages (but maybe hemorrhages is the American spelling?)
Preformance Peaker's "name =" is spelled incorrectly. (also it's performance, not preformance)
/obj/item/weapon/reagent_containers/food/snacks/metroidmeat has the name "Metroid Hude", is it supposed to be "Metroid Hide"?


These were all the corrections, suggestions, and questions relating to the code that I could find.
If you'd like, I can make suggested commit changes myself if this gets merged as a quick-fix if you'd rather not worry about the grammar yourself. I'm not going to ask for access to your branch so I can instead offer to do a hotfix for you, just ping me on Discord or something if this gets merged. <3

**Comment by FoxglovePurpur** [comment]
Same as on Discord, I go by fox, foxglove, foxglovepur, or foxglovepurpur everywhere I go. :3  (And ofc <3)

**Comment by xnqm8h** [comment]
The only one I might have issue with atm is Souldew. We have necroxadone at the moment and that effects corpses but requires a more elaborate setup. Though it only works for toxins, oxy, and clone damage, which is almost the same thing you're trying to implement here it seems.

See:
https://github.com/CHOMPStation2/CHOMPStation2/blob/38587b50dd1a4ecdffd32629afe4cbcd553fa5a3/code/modules/reagents/reagents/medicine.dm#L456-L487

**Comment by Raeschen** [comment]
consider rebasing this to master.

File: modular_chomp/code/modules/mob/living/simple_mob/butchering.dm [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//I don't know how to add more material currently so uhhh, that will be for later.
+/mob/living/simple_mob/vore/bigdragon
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/dragonmeat
+
+	//butchery_loot = list(\
```
Comment (Raeschen) [inline modular_chomp/code/modules/mob/living/simple_mob/butchering.dm:5]: not sure what the backslashes here are for.
Block comment with /* */ if that's your intention

Comment (FluffMedic) [reply modular_chomp/code/modules/mob/living/simple_mob/butchering.dm]: Pretty much was trying to set up the groundworks for a future PR, but seeing as what I attempted is causing issue removed that it. The //butchery_loot.

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 2]
```diff
@@ -23,3 +23,105 @@
...
+
+//Chemist update
+/decl/chemical_reaction/instant/livingagent
+	name = "Living Agent"
+	id = "livingagent"
+	result = "livingagent"
+	required_reagents = list("tricordrazine" = 1, "liquidlife" = 1, "sugar" = 1)
+	result_amount = 2
+
+/decl/chemical_reaction/instant/preformancepeaker
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:35]: Do you mean 'performance' over 'preformance'?

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 3]
```diff
@@ -23,3 +23,105 @@
...
+	name = "Preforamnce Peaker"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:36]: 'preforamnce'--> 'preformance'----> 'performance'?

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 4]
```diff
@@ -23,3 +23,105 @@
...
+	result_amount = 2
+
+/decl/chemical_reaction/instant/bloodsealer
+	name = "Blood Sealer"
+	id = "bloodsealer"
+	result = "bloodsealer"
+	required_reagents = list("neoliquidfire" = 1, "glucose" = 1, "hydrogen" = 1)
+	result_amount = 2
+
+/decl/chemical_reaction/instant/puryfingagent
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:70]: puryfing --> purifying?

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 5]
```diff
@@ -23,3 +23,105 @@
...
+
+//tier 3
+/decl/chemical_reaction/instant/liquidangel
+	name = "liquidangel"
+	id = "liquidangel"
+	result = "liquidangel"
+	required_reagents = list("bicard" = 1, "kelotane" = 1, "liquidlife" = 1, "carpotoxin" = 1, "quadcord" = 1, "cryptobiolin" = 1)
+	result_amount = 6
+
+/decl/chemical_reaction/instant/phoneixbreath
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:122]: phoneix --> phoenix?

File: modular_chomp/code/modules/reagents/reagents/medicine.dm [hunk 6]
```diff
@@ -62,3 +62,301 @@
 				M.change_gender_identity(gender_change)
 				M.change_gender(gender_change)
 				M << "<span class='warning'>You feel like a new person.</span>" //success
+
+//Chemist expansion
+//deathblood
+/datum/reagent/cleansingagent
+	name = "Puryfing Agent"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reagents/medicine.dm:69]: copypaste error 'puryfing' this is for Cleansing agent

File: modular_chomp/code/modules/reagents/reagents/medicine.dm [hunk 7]
```diff
@@ -62,3 +62,301 @@
...
+	M.AdjustWeakened(-1)
+	M.add_chemical_effect(CE_PAINKILLER, 10 * M.species.chem_strength_pain)
+	M.adjustToxLoss(15 * removed)
+
+//advanced crafting
+//tier 1
+/datum/reagent/souldew
+	name = "Soul Dew"
+	id = "souldew"
+	description = "An expirmental drug that soley works upon dead bodies"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reagents/medicine.dm:258]: soley --> solely

File: modular_chomp/code/modules/reagents/reagents/medicine.dm [hunk 8]
```diff
@@ -62,3 +62,301 @@
...
+
+/datum/reagent/juggernog/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	M.AdjustParalysis(-1)
+	M.AdjustStunned(-1)
+	M.AdjustWeakened(-1)
+
+/datum/reagent/curea
+	name = "curea"
+	id = "curea"
+	description = "An experimental that removes many aligments, such as poison and stiffing of muscles via frost"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reagents/medicine.dm:311]: aligments -> ailments
stiffing -> stiffening

Comment (Raeschen) [reply modular_chomp/code/modules/reagents/reagents/medicine.dm:311]: 'An experimental that..' --> 'an experimental drug that...'

File: modular_chomp/code/modules/reagents/reagents/medicine.dm [hunk 9]
```diff
@@ -62,3 +62,301 @@
...
+	M.remove_a_modifier_of_type(/datum/modifier/grievous_wounds)
+	M.remove_a_modifier_of_type(/datum/modifier/deep_wounds)
+	M.remove_a_modifier_of_type(/datum/modifier/hivebot_weaken)
+	M.remove_a_modifier_of_type(/datum/modifier/fire)
+
+//tier 3
+/datum/reagent/modapplying/liquidangel
+	name = "Liquid Healer"
+	id = "liquidangel"
+	description = "An experimental that mimics rapid regeneration seen in squishy creatures."
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reagents/medicine.dm:333]: 'An experimental that..' --> 'an experimental drug that...'

File: modular_chomp/code/modules/reagents/reagents/toxin.dm [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+	description = "Blood of dragon."
+	taste_description = "plastic"
+	reagent_state = LIQUID
+	color = "#CF3600"
+	strength = 8
+
+/datum/reagent/toxin/liquidlife
+	name = "Liquid Life"
+	id = "liquidlife"
+	description = "Life Drained from a victim, and processed into a liquid form, ."
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reagents/toxin.dm:32]: "form, ." --> "form."

**Comment by FluffMedic** [comment]
Agg, need to go and spell fix the medkit

File: modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+	icon = 'modular_chomp/icons/obj/storage.dmi'
+	icon_state = "expirmentalaid"
+	starts_with = list(
+		/obj/item/weapon/storage/pill_bottle/neotane,
+		/obj/item/weapon/storage/pill_bottle/burncard,
+		/obj/item/weapon/storage/pill_bottle/flamecure,
+		/obj/item/weapon/storage/pill_bottle/juggernog,
+		/obj/item/weapon/storage/pill_bottle/curea,
+		/obj/item/weapon/storage/pill_bottle/souldew,
+		/obj/item/weapon/storage/pill_bottle/puriyfingagent)
```
Comment (Raeschen) [inline modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm:13]:
```suggestion
		/obj/item/weapon/storage/pill_bottle/purifyingagent)

```

File: modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+	starts_with = list(/obj/item/weapon/reagent_containers/pill/curea = 7)
+	wrapper_color = COLOR_BLUE
+
+/obj/item/weapon/storage/pill_bottle/souldew
+	name = "pill bottle (soul dew)"
+	desc = "Contains experimental pills."
+	starts_with = list(/obj/item/weapon/reagent_containers/pill/souldew = 7)
+	wrapper_color = COLOR_GREEN
+
+/obj/item/weapon/storage/pill_bottle/puriyfingagent
```
Comment (Raeschen) [inline modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm:51]:
```suggestion
/obj/item/weapon/storage/pill_bottle/purifyingagent

```

File: modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+	name = "pill bottle (puriyfing agent)"
```
Comment (Raeschen) [inline modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm:52]:
```suggestion
	name = "pill bottle (purifying agent)"

```

File: modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm [hunk 14]
```diff
@@ -0,0 +1,126 @@
...
+	desc = "Contains experimental pills."
+	starts_with = list(/obj/item/weapon/reagent_containers/pill/puriyfingagent = 7)
```
Comment (Raeschen) [inline modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm:54]:
```suggestion
	starts_with = list(/obj/item/weapon/reagent_containers/pill/purifyingagent = 7)

```

File: modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm [hunk 15]
```diff
@@ -0,0 +1,126 @@
...
+	wrapper_color = COLOR_GREEN
+
+//Pills
+/obj/item/weapon/reagent_containers/pill/neotane
+	name = "Neo Kelotane (15u)"
+	desc = "An expirmental pill."
```
Comment (Raeschen) [inline modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm:60]:
```suggestion
	desc = "An experimental pill."

```

File: modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm [hunk 16]
```diff
@@ -0,0 +1,126 @@
...
+	name = "Souldew (10u)"
+	desc = "An expirmental pill."
+	icon_state = "pill2"
+
+/obj/item/weapon/reagent_containers/pill/souldew/Initialize()
+	. = ..()
+	reagents.add_reagent("souldew", 10)
+	color = reagents.get_color()
+
+/obj/item/weapon/reagent_containers/pill/puriyfingagent
```
Comment (Raeschen) [inline modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm:118]:
```suggestion
/obj/item/weapon/reagent_containers/pill/purifyingagent
```

File: modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm [hunk 17]
```diff
@@ -0,0 +1,126 @@
...
+	name = "Puriyfing Agent (10u)"
```
Comment (Raeschen) [inline modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm:119]:
```suggestion
	name = "Purifying Agent (10u)"

```

File: modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm [hunk 18]
```diff
@@ -0,0 +1,126 @@
...
+	desc = "An expirmental pill."
+	icon_state = "pill2"
+
+/obj/item/weapon/reagent_containers/pill/puriyfingagent/Initialize()
```
Comment (Raeschen) [inline modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm:123]:
```suggestion
/obj/item/weapon/reagent_containers/pill/purifyingagent/Initialize()

```

File: modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm [hunk 19]
```diff
@@ -0,0 +1,126 @@
...
+	. = ..()
+	reagents.add_reagent("puriyfingagent", 10)
```
Comment (Raeschen) [inline modular_chomp/code/game/objects/items/weapons/storage/firstaid.dm:125]:
```suggestion
	reagents.add_reagent("purifyingagent", 10)

```

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 20]
```diff
@@ -23,3 +23,105 @@
...
+	result_amount = 2
+
+/decl/chemical_reaction/instant/bloodsealer
+	name = "Blood Sealer"
+	id = "bloodsealer"
+	result = "bloodsealer"
+	required_reagents = list("neoliquidfire" = 1, "glucose" = 1, "hydrogen" = 1)
+	result_amount = 2
+
+/decl/chemical_reaction/instant/puriyfingagent
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:70]:
```suggestion
/decl/chemical_reaction/instant/purifyingagent
```

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 21]
```diff
@@ -23,3 +23,105 @@
...
+	name = "Puriyfing Agent"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:71]:
```suggestion
	name = "Purifying Agent"
```

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 22]
```diff
@@ -23,3 +23,105 @@
...
+	id = "puriyfingagent"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:72]:
```suggestion
	id = "purifyingagent"
```

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 23]
```diff
@@ -23,3 +23,105 @@
...
+	result = "puryfingagent"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:73]:
```suggestion
	result = "purifyingagent"
```

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 24]
```diff
@@ -23,3 +23,105 @@
...
+
+//tier 3
+/decl/chemical_reaction/instant/liquidangel
+	name = "liquidangel"
+	id = "liquidangel"
+	result = "liquidangel"
+	required_reagents = list("bicard" = 1, "kelotane" = 1, "liquidlife" = 1, "carpotoxin" = 1, "quadcord" = 1, "cryptobiolin" = 1)
+	result_amount = 6
+
+/decl/chemical_reaction/instant/phoneixbreath
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:122]:
```suggestion
/decl/chemical_reaction/instant/phoenixbreath
```

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 25]
```diff
@@ -23,3 +23,105 @@
...
+	name = "phoneixbreath"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:123]:
```suggestion
	name = "phoenixbreath"
```

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 26]
```diff
@@ -23,3 +23,105 @@
...
+	id = "phoneixbreath"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:124]:
```suggestion
	id = "phoenixbreath"
```

File: modular_chomp/code/modules/reagents/reactions/instant/instant.dm [hunk 27]
```diff
@@ -23,3 +23,105 @@
...
+	result = "phoneixbreath"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reactions/instant/instant.dm:125]:
```suggestion
	result = "phoenixbreath"
```

File: modular_chomp/code/modules/reagents/reagents/medicine.dm [hunk 28]
```diff
@@ -62,3 +62,301 @@
...
+		chem_effective = 0.66
+	if(alien != IS_DIONA)
+		M.druggy = max(M.druggy, 5)
+		M.adjustToxLoss(-6 * removed * chem_effective)
+		M.radiation = max(M.radiation - 15 * removed * M.species.chem_strength_heal, 0)
+		M.accumulated_rads = max(M.accumulated_rads - 15 * removed * M.species.chem_strength_heal, 0)
+
+/datum/reagent/purifyingagent
+	name = "Purifying Agent"
+	id = "puriyfingagent"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reagents/medicine.dm:90]:
```suggestion
	id = "purifyingagent"
```

File: modular_chomp/code/modules/reagents/reagents/medicine.dm [hunk 29]
```diff
@@ -62,3 +62,301 @@
...
+				if(W.internal)
+					W.damage = max(W.damage - wound_heal, 0)
+					if(W.damage <= 0)
+						O.wounds -= W
+
+//neoliquidfire
+/datum/reagent/neotane
+	name = "Neo Kelotane"
+	id = "neotane"
+	description = "An advancment of kelotane that scars and breaks apart the users flesh to remove the burnt tissue."
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reagents/medicine.dm:178]:
```suggestion
	description = "An advancement of kelotane that scars and breaks apart the user's flesh to remove the burnt tissue."
```

File: modular_chomp/code/modules/reagents/reagents/medicine.dm [hunk 30]
```diff
@@ -62,3 +62,301 @@
...
+
+/datum/reagent/juggernog/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	M.AdjustParalysis(-1)
+	M.AdjustStunned(-1)
+	M.AdjustWeakened(-1)
+
+/datum/reagent/curea
+	name = "curea"
+	id = "curea"
+	description = "An experimental that removes many aligments, such as poison and stiffing of muscles via frost"
```
Comment (Raeschen) [inline modular_chomp/code/modules/reagents/reagents/medicine.dm:311]:
```suggestion
	description = "An experimental that removes many ailments, such as poison and stiffening of muscles via frost"
```

**Comment by Raeschen** [comment]
there's also numerous instances of misspelling the word 'Experimental.' 
'expirmental' was used.

**Pull request merged**